site/ru/README.md

```diff
@@ -1,25 +1,30 @@
-# Не Официальный Перевод
+# Неофициальный перевод
 
-Вся информация в этом разделе переведена при помощи русскоговорящего Tensorflow
-сообщества на общественных началах. По скольку этот перевод не является
+Вся информация в этом разделе переведена с помощью русскоговорящего Tensorflow
+сообщества на общественных началах. Поскольку этот перевод не является
 официальным, мы не гарантируем что он на 100% аккуратен и соответствует
 [официальной документации на английском языке](https://www.tensorflow.org/?hl=en).
-Если у вас есть предложение как исправить этот перевод, мы будем очень рады
+
+1. Если у вас есть предложение как исправить этот перевод, мы будем очень рады
 увидеть pull request в [tensorflow/docs](https://github.com/tensorflow/docs)
-GitHub репозиторий. Если вы хотите помочь сделать документацию по Tensorflow
-лучше (сделать сам перевод или проверить перевод подготовленный кем то другим),
+репозиторий GitHub. 
+
+2. Если вы хотите помочь сделать документацию по Tensorflow
+лучше (сделать сам перевод или проверить перевод подготовленный кем-то другим),
 заполните [эту форму](https://bit.ly/tf-translate) или напишите нам на
 [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
 
 # Community translations
 
-Our TensorFlow community has translated these documents. Because community
+These docs were translated by our TensorFlow community. Because community
 translations are *best-effort*, there is no guarantee that this is an accurate
 and up-to-date reflection of the
-[official English documentation](https://www.tensorflow.org/?hl=en). If you have
-suggestions to improve this translation, please send a pull request to the
-[tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository. To
-volunteer to write or review community translations, fill out
+[official English documentation](https://www.tensorflow.org/?hl=en). 
+
+1. If you have suggestions to improve this translation, please send a pull request 
+to the [tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository. 
+
+2. To volunteer to write or review community translations, fill out
```

Comment:
Regarding your other comment, the form is a little early. Can you change this line (and the translations) to:

```
2. To volunteer to write or review community translations, contact the [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
```